### ISSUE TITLE
Mouse scroll down does not work in VS Code (scroll up works)

### ISSUE THREAD

**Issue by Farkuldi** [microsoft/vscode#88252]
It has been a while since I last used VS Code, and I just reinstalled it. And I am greeted by a weird issue where I cannot scroll down with my Logitech external mouse using my MacBook Pro. The trackpad does work as expected. The mouse works in all other applications as expected. And strangest of all, it works in VS Code if I try to scroll up the page--just not down.

I actually have multiple versions of Code around due to Mac's Time Machine, and I tried three different versions and found the same issue. I always used the trackpad before when working and only recently moved to a large external monitor with a mouse, so I may just never have noticed this. Or it is possible a new OS update caused it. Tried with extensions disabled and got the same thing. 

- VSCode Version: 1.41.1
- OS Version: MacOS 10.14.6

Steps to Reproduce:

1. Use external mouse, not trackpad. 
2. Attempt to scroll down. Doesn't work. 
3. Manually drag the slider down the page to get to a place where you can scroll up, then try to do so. Works.

Does this issue occur when all extensions are disabled?: Yes

**Comment by connor4312** [comment]
Can you share your settings and any installed extensions? Also, do you have this same issue with any other programs on your machine?

/needsMoreInfo

**Comment by Farkuldi** [comment]
I have tried it with absolutely no extensions and the out of the box settings.json. When I reinstalled,  I deleted the code folder in Application Support and let it create a new one, so I would think it would be a clean reinstall I also used 'code --no-extensions'. Also no, as I mentioned above, the mouse works perfectly in all other applications.

**Comment by Farkuldi** [comment]
I checked the developer console and I have one error:

`/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:1572   ERR No application in the Launch Services database matches the input criteria.: Error: No application in the Launch Services database matches the input criteria.`

**Comment by connor4312** [comment]
Okay, apologies for the duplication, I had opened your issue and was interrupted while reading it 🙂 

Does scrolling work in _any_ area inside VS code. For instance, is it just the code editor where scrolling is broken, or do you have the some problem in the file tree view or the settings editor (ctrl+shift+p -> user settings)?

**Comment by Farkuldi** [comment]
Thanks, no worries. :) I just tried the file tree view and the settings editor (ctrl + shift + p), and it is the same. So it seems to be everywhere.

**Comment by Farkuldi** [comment]
Yes, I mentioned above that I tried with another mouse and it was the same. One was a Logitech Anywhere 2 mouse, and the other was a basic cheap Logitech mouse. The Anywhere 2 mouse might have some customizable drivers available for Windows, but I do not have them installed on my Mac. I'm not sure they are even available on Mac. The other mouse has no such drivers. I am using a tool called Better Touch Tool that can add custom UI events like you mention, but I did not configure any for the mouse, and I also disabled the tool before posting this issue just in case it was the culprit. The behavior was the same.

However, the problem seems to have spontaneously resolved. I tried with the mouse again and it works now. I have rebooted several times (which I also did multiple times before posting this issue, for troubleshooting), but I have done no OS updates. I am using the same mice as before. I have no idea what happened--only that it was software related somehow, since the mice worked fine in other applications. Maybe it was an OS issue that was fixed on the tenth reboot.

I promise the problem existed, and was persistent for over a week of rebooting and rechecking. Very strange!
